.coveragerc

```diff
@@ -1,4 +1,6 @@
 [run]
+omit =
+    robottelo/commands/*.py
```

Comment:
This is here because we can't test commands, it would demand an environment configured to run the commands, also the command should only be a starting point to existing helpers and features so lets test that not commands.